proprietary/tokens/src/components/amsterdam/top-task-link.tokens.json

```diff
@@ -2,33 +2,61 @@
   "amsterdam": {
     "top-task-link": {
       "description": {
-        "color": { "value": "{amsterdam.color.primary-black}" },
-        "font-family": { "value": "{amsterdam.typography.font-family}" },
-        "font-weight": { "value": "{amsterdam.typography.font-weight.normal}" },
-        "line-height": { "value": "{amsterdam.typography.text-level.7.line-height}" },
+        "color": {
```

Comment:
Do you have local prettier settings that differ from [our prettier config](https://github.com/Amsterdam/design-system/blob/main/.prettierrc.json)? I don't think these line breaks are necessary?